kgraphql/src/main/kotlin/com/apurebase/kgraphql/schema/structure/RequestInterpreter.kt

```diff
@@ -57,9 +57,9 @@ class RequestInterpreter(private val schemaModel: SchemaModel) {
         variables: VariablesJson,
         options: ExecutionOptions
     ): ExecutionPlan {
-        val test = document.definitions.filterIsInstance<ExecutableDefinitionNode>()
+        val executables = document.definitions.filterIsInstance<ExecutableDefinitionNode>()
 
-        val operation = test.filterIsInstance<OperationDefinitionNode>().let { operations ->
+        val operation = executables.filterIsInstance<OperationDefinitionNode>().let { operations ->
```

Comment:
Performance:

Is there any reason why we filter those twice instead of searching for exactly the wanted entries in the first place?

Reply:
`executables` is filtered for a different instance later in the function.